.github/workflows/lint.yml

```diff
@@ -0,0 +1,13 @@
+name: Lint
+on: [push, pull_request]
```

Comment:
I've been doing scheduled builds, once a week, for my recent Actions. Thoughts about doing it here too? We have something similar set up in Travis, controlled in the web UI.

The motivation is to discover if the project is failing to build due to tooling or dep changes so that it can be fixed before someone makes a PR.